src/world/leveldb/Leveldb.ts

```diff
@@ -65,6 +65,8 @@ export default class LevelDB extends Provider {
                             const subChunkVersion = stream.readByte();
                             if (subChunkVersion == 0) {
                                 let blocks = stream.read(4096);
+
+                                // TODO: This should be read as halfbytes instead of bytes
```

Comment:
This is also the reason for this:
https://github.com/JSPrismarine/JSPrismarine/blob/master/src/world/chunk/SubChunk.ts#L13
https://github.com/JSPrismarine/JSPrismarine/blob/master/src/world/chunk/SubChunk.ts#L42